pt/lessons/advanced/concurrency.md

```diff
@@ -1,19 +1,19 @@
 ---
-version: 1.1.1
+version: 1.1.2
```

Comment:
```suggestion
version: 1.1.1
```
@elainenaomi Even though this is a improvement to the translation we should keep the same version as the original https://github.com/elixirschool/elixirschool/blob/master/en/lessons/advanced/concurrency.md. We only bump when the content from the original changes.

Reply:
Oh! Ok! Thanks!